std_msgs/Header header

bool qr_update
int16 qr_center_x
int16 qr_center_y
int16 qr_height
int16 qr_width

bool logo_update
int16 logo_center_x
int16 logo_center_y
int16 logo_height
int16 logo_width

